kpatch-build/kpatch-build

```diff
@@ -97,24 +97,27 @@ find_dirs() {
 		# git repo
 		TOOLSDIR="$SCRIPTDIR"
 		DATADIR="$(readlink -f $SCRIPTDIR/../kmod)"
-		SYMVERSFILE="$DATADIR/core/Module.symvers"
-		return
 	elif [[ -e "$SCRIPTDIR/../libexec/kpatch/create-diff-object" ]]; then
 		# installation path
 		TOOLSDIR="$(readlink -f $SCRIPTDIR/../libexec/kpatch)"
 		DATADIR="$(readlink -f $SCRIPTDIR/../share/kpatch)"
+	fi
+}
+
```

Comment:
Let's have `find_dirs()` return an error if it could not find `create-diff-object` in either location.
Then later when `find_dirs` is invoked, we can check for the error: 
`find_dirs || die "can't find supporting tools"`